README.md

```diff
@@ -43,4 +43,8 @@ or libc++abi.a, respectively, so they may not be usable without
 extra setup. This is one of the things [wasi-sdk] simplifies, as it includes
 cross-compiled builds of compiler-rt, libc++.a, and libc++abi.a.
 
+## AUR package
```

Comment:
Perhaps put "Arch Linux" in the heading here.. since its more meaningful that "AUR package"... I had to look up that the latter was.

You can also ways say something like "For Arch Linux there is a unofficial (AUR) package..."